dbt_style_guide.md

```diff
@@ -148,18 +263,23 @@ some_cte_agg as (
 final as (
 
     select [distinct]
+

```

Comment:
This appears to be a change. I used to do this and then had to train myself out of it. I actually kind of like having the next row be right below select but I can understand why others might like it this way.

Reply:
Agree with Sam. In most cases I don't think extra spacing in the select statement makes it any more readable unless it's a large calculation or case statement. And in most of those cases as long as you utilize multiple lines and indent it's still very readable.

Reply:
No strong preference on this one from me!

Reply:
Same - the reason I changed it was that we are add white space at the bottom of the select statement (before the from) and also in between our CTE `()` and the selects within, so changing this made it consistent with the other styling we do. Either way is not a big deal to me, but I think the consistency should show in all the places we do this. For example:

```sql
with

some_item as (
    select * from {{ ref('some_thing') }}
),

other_item as (
    select * from {{ ref('some_other_thing') }}
),

joined as (
    select
        some_item.some_column,
        other_item.some_other_column
    from some_item
    left join other_item
        on some_item.item_id = other_item.item_id
)

select * from joined
```

vs

```sql
with

some_item as (

    select * from {{ ref('some_thing') }}

),

other_item as (

    select * from {{ ref('some_other_thing') }}

),

joined as (

    select

        some_item.some_column,
        other_item.some_other_column

    from some_item
    left join other_item
        on some_item.item_id = other_item.item_id

)

select * from joined
```

Reply:
I personally vote #1 of Christine's comment example. But will be ok with the extra whitespace!

Reply:
As I said in my previous comment, I have no strong preference on this but for voting's sake I'll choose #1 because:
1. This is how it's laid out throughout the guide and I don't feel like changing it. 😆 
2. In my experience the whitespace does add a breath of fresh air when you get into really big models with lengthy code.

Reply:
i'll vote for #2!